S/shaderc/build_tarballs.jl

```diff
@@ -0,0 +1,54 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "shaderc"
+version = v"2022.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/google/shaderc.git", "e4722b0ad49ee60c143d43baae8390f75ba27d2d")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/shaderc*/
+
+./utils/git-sync-deps 
+
+mkdir build && cd build
+
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DSHADERC_SKIP_TESTS=1 -DSHADERC_SKIP_EXAMPLES=1 -DSHADERC_SKIP_COPYRIGHT_CHECK=1 -S .. -B .
+
+make -j${nproc}
+
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
```

Comment:
From the log:
```
┌ Warning: /agent/_work/1/s/S/shaderc/build/i686-linux-gnu/Mjo17nMt/i686-linux-gnu-libgfortran5-cxx11/destdir/bin/glslangValidator contains std::string values!  This causes incompatibilities across the GCC 4/5 version boundary.  To remedy this, you must build a tarball for both GCC 4 and GCC 5.  To do this, immediately after your `platforms` definition in your `build_tarballs.jl` file, add the line:
│ 
│     platforms = expand_cxxstring_abis(platforms)
└ @ BinaryBuilder.Auditor /depot/packages/BinaryBuilder/Ke46L/src/auditor/compiler_abi.jl:247
```suggestion
platforms = expand_cxxstring_abis(supported_platforms())
```
```

Reply:
ok